neo/test/rawiotest/test_blackrockrawio.py

```diff
@@ -184,6 +185,32 @@ def test_compare_blackrockio_with_matlabloader_v21(self):
                         matlab_digievents = mts_ml[mid_ml == int(label)]
                         assert_equal(python_digievents, matlab_digievents)
 
+    def test_blackrockrawio_ptp_timestamps(self):
+        dirname = self.get_local_path("blackrock/blackrock_3_0_ptp/20231027-125608-001")
```

Comment:
```suggestion
        dirname = self.get_local_path("blackrock/blackrock_3_0_ptp")
```

It looks to me like Sam's PR of your data removed this part of the file so the folder structure is just without the ending part.